frontend/src/views/ElectedCanteens.vue

```diff
@@ -0,0 +1,150 @@
+          Publiée
+        </router-link>
+        <span v-else>
+          Non publiée
+        </span>
+      </template>
+    </v-data-table>
+  </div>
+</template>
+
+<script>
+import jsonDepartments from "@/departments.json"
+import { getObjectDiff } from "@/utils"
+import BreadcrumbsNav from "@/components/BreadcrumbsNav"
+
+export default {
+  name: "ElectedCanteens",
+  components: { BreadcrumbsNav },
+  data() {
+    return {
+      loading: false,
+      canteenCount: null,
+      visibleCanteens: [],
+      limit: 10,
+      options: {
+        sortBy: [],
+        sortDesc: [],
+        page: 1,
+      },
+      footerProps: {
+        disableItemsPerPage: true,
+        itemsPerPageText: "Cantines per page",
+      },
+      headers: [
+        { text: "SIRET", value: "siret", sortable: true },
+        { text: "Nom", value: "name", sortable: true },
+        { text: "Ville", value: "city", sortable: true },
+        { text: "Couverts moyen par jour", value: "dailyMealCount", sortable: true },
+        { text: "Publication", value: "publicationStatus", sortable: false },
+      ],
+    }
+  },
+  computed: {
+    offset() {
+      return (this.options.page - 1) * this.limit
+    },
+    departmentsString() {
+      const userDepartments = this.$store.state.loggedUser.departments
+      const departmentsArray = jsonDepartments
+        .filter((x) => userDepartments.indexOf(x.departmentCode) > -1)
+        .map((x) => `${x.departmentName} (${x.departmentCode})`)
+      if (departmentsArray.length < 2) return `le département ${departmentsArray.join("")}`
+      return `les départements ${departmentsArray.slice(0, -1).join(", ")} et ${departmentsArray.slice(-1)}`
+    },
+  },
+  methods: {
+    fetchCurrentPage() {
+      this.loading = true
+      return fetch(`/api/v1/electedCanteens/?${this.getApiQueryParams()}`)
+        .then((response) => {
+          if (response.status < 200 || response.status >= 400) throw new Error(`Error encountered : ${response}`)
+          return response.json()
+        })
+        .then((response) => {
+          this.visibleCanteens = response.results
+          this.canteenCount = response.count
+        })
+        .catch((e) => {
+          this.publishedCanteenCount = 0
+          this.visibleCanteens = []
+          this.$store.dispatch("notifyServerError", e)
+        })
+        .finally(() => (this.loading = false))
+    },
+    getApiQueryParams(includePagination = true) {
+      let apiQueryParams = includePagination ? `limit=${this.limit}&offset=${this.offset}` : ""
+      const orderingItems = this.getOrderingItems()
+      if (orderingItems.length > 0) apiQueryParams += `&ordering=${orderingItems.join(",")}`
+      return apiQueryParams
+    },
+    addWatchers() {
+      this.$watch("options", this.onOptionsChange, { deep: true })
+      this.$watch("$route", this.onRouteChange)
+    },
+    onOptionsChange() {
+      this.$router.push({ query: this.getUrlQueryParams() }).catch(() => {})
```

Comment:
I would be interested to know whether changing search order is really something that users like having in their search history or not. I think it's important for filters and searches, but I'm not so certain in this case. Nothing needs to be changed atm I think, since this is consistent across our site, but might be worth asking Arthur for a second opinion.

Reply:
Agree. It's a broader discussion since it might also be present in other places of the app